tests/testthat/test-modify-list.R

```diff
@@ -0,0 +1,25 @@
+context("modify_list")
+
+testlist <- list(
+  a = 5.5,
+  b = "x",
+  c = 1:10
+)
+testappend <- list(
+  b = "y",
+  c = NULL,
+  d = FALSE
+)
+
+test_that("modifyList is masked", {
+  expect_error(modifyList(testlist, testappend))
+})
+
+test_that("modify_list does what it should", {
```

Comment:
I think this would be clearer broken out into separate tests like "modify_list() removes NULL elements"